### PR TITLE
git-aliases: Add version 0.3.5

### PR DESCRIPTION
**Pull request by sitiom**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/git-aliases.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    ],
+    "url": "https://github.com/gluons/powershell-git-aliases/archive/v0.3.5.zip",
+    "hash": "6f51ffcb968de60ecd434bd5074726e6a8818713cf86821782c9d1ffd39b7ef5",
+    "extract_dir": "powershell-git-aliases-0.3.5\\src",
+    "psmodule": {
+        "name": "git-aliases"
+    },
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/dahlbyk/posh-git/archive/v$version.zip",
```
Comment (niheaven) [inline bucket/git-aliases.json:18]: Wrong URL

**Comment by niheaven** [comment]
Why not use nupkg?

https://psg-prod-eastus.azureedge.net/packages/git-aliases.0.3.5.nupkg

**Comment by sitiom** [comment]
> Why not use nupkg?
> 
> https://psg-prod-eastus.azureedge.net/packages/git-aliases.0.3.5.nupkg

Direct to source if possible. I also see [`posh-git`](https://github.com/ScoopInstaller/Extras/blob/master/bucket/posh-git.json) doing it.

**Comment by niheaven** [comment]
`posh-git` is created before added to PSGallery, so source is used. Use a more stable and accessible host is preferred.

**Comment by rashil2000** [comment]
A `suggest` for git should be added

**Comment by sitiom** [comment]
> `posh-git` is created before added to PSGallery, so source is used. Use a more stable and accessible host is preferred.

I don't think so, posh-git has been in PSGallery since the beginning.

**Comment by sitiom** [comment]
Switched to using PSGallery.

**Review by niheaven — APPROVED** [review]

**Pull request merged**